packages/core/src/App/Components/Elements/CookieBanner/cookie-banner.jsx

```diff
@@ -5,11 +5,11 @@ import { Button } from '@deriv/components';
 import { getDerivComLink } from '@deriv/shared/utils/url';
 import { localize, Localize } from '@deriv/translations';
 
-const CookieBanner = ({ onAccept, onDecline, is_open }) => (
+const CookieBanner = ({ onAccept, onDecline, is_open, is_dark_mode }) => (
     <div
         className={classNames('cookie-banner', {
-            'cookie-banner--is-open': is_open,
```

Comment:
Just a question for my own understanding:

So there's no `cookie-banner--is-open` class styles defined anywhere, right?

Reply:
Yes.